### PR TITLE
[IMM32] ImmDisableIME calls NtUserDisableThreadIme

### PR DESCRIPTION
**Pull request by katahiromz**
## Purpose
I have started development of imm32-related stuffs.
JIRA issue: [CORE-11700](https://jira.reactos.org/browse/CORE-11700)

## Proposed changes

- Just call `NtUserDisableThreadIme` in `ImmDisableIME` directly.

### CODE REVIEW

**Comment by katahiromz** [comment]
EVIDENCE:
https://github.com/katahiromz/secret_base/blob/0a5e0e23a55494a8a14524ce54ab687ba5a8fee6/imm32/analysis/Func76292476.txt

File: dll/win32/imm32/imm.c [hunk 1]
```diff
@@ -850,6 +850,9 @@ BOOL WINAPI ImmDestroyContext(HIMC hIMC)
  */
 BOOL WINAPI ImmDisableIME(DWORD idThread)
 {
+#ifdef __REACTOS__
+    return NtUserDisableThreadIme(idThread); /* confirmed */
```
Comment (learn-more) [inline dll/win32/imm32/imm.c:854]: The 'confirmed' comment is not needed in my opinion.

**Review by learn-more — APPROVED** [review]
lgtm.

If our ime support is going to be implemented in win32k, it makes sense to fork this imm32 module completely (e.g. drop #ifdef)

File: dll/win32/imm32/imm.c (hunk 1 above)
Comment (katahiromz) [reply dll/win32/imm32/imm.c]: Deleted the comment.

**Comment by katahiromz** [comment]
Check time. I will merge 1 day later.

**Pull request merged**